keyboards/nullbitsco/nibble/keymaps/default/keymap.c

```diff
@@ -0,0 +1,81 @@
+/* Copyright 2018 Jack Humbert
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+#include "remote_kb.h" 
```

Comment:
This should not be included here.  If this is needed for the keymap to function properly, it needs to be included in the keyboard's header file, instead. 

```suggestion
```